Tests/AfterTraining/test_after_training.py

```diff
@@ -90,6 +91,26 @@ def test_model_file_structure(test_output_dirs: OutputFolderForTests) -> None:
         pytest.fail(f"{len(missing)} files in the registered model are missing: {missing[:5]}")
 
 
+@pytest.mark.after_training
+def test_get_comparison_data(test_output_dirs: OutputFolderForTests) -> None:
```

Comment:
this test is a good start, but prone to run out of sync with the actual code. Can you refactor `get_comparison_baselines` such that you can test the downloading of files in isolation?